[2024-02-29T21:34:10.716+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task manual__2024-02-29T21:34:08.796180+00:00 [queued]>
[2024-02-29T21:34:10.721+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task manual__2024-02-29T21:34:08.796180+00:00 [queued]>
[2024-02-29T21:34:10.722+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-02-29T21:34:10.733+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): third_task> on 2024-02-29 21:34:08.796180+00:00
[2024-02-29T21:34:10.738+0000] {standard_task_runner.py:60} INFO - Started process 2081 to run task
[2024-02-29T21:34:10.745+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'manual__2024-02-29T21:34:08.796180+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpmxxtr5n6']
[2024-02-29T21:34:10.749+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask third_task
[2024-02-29T21:34:10.925+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.third_task manual__2024-02-29T21:34:08.796180+00:00 [running]> on host 417e643aed36
[2024-02-29T21:34:13.402+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dn' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-29T21:34:08.796180+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-29T21:34:08.796180+00:00'
[2024-02-29T21:34:13.407+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-29T21:34:13.410+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and will be running after task1 at the same time as task2']
[2024-02-29T21:34:13.428+0000] {subprocess.py:86} INFO - Output:
[2024-02-29T21:34:13.431+0000] {subprocess.py:93} INFO - hey, I am task3 and will be running after task1 at the same time as task2
[2024-02-29T21:34:13.432+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-29T21:34:13.473+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, execution_date=20240229T213408, start_date=20240229T213410, end_date=20240229T213413
[2024-02-29T21:34:13.585+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-29T21:34:13.641+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
